src/packer.pkr.hcl

```diff
@@ -80,9 +80,9 @@ locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
-    device_name           = "/dev/xvda"
+    device_name           = "/dev/sda1"
```

Comment:
Why was this device name changed? There's no explanation in the commit the change took place.

Reply:
I changed the device name while debugging early on in this PR. After some research, I now understand why `/dev/xvda` is the correct device_name.